### ISSUE TITLE
NTR: Pancreatic biopsy: VIPoma (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9149]
New term request Pancreatic biopsy: VIPoma (elthat_100321055039): Vasoactive intestinal peptide-secreting  UMLS: 4030390

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000897
- label: Pancreatic VIPoma
- parents: [Neoplasm of the pancreas (HP:0002894)](https://hpo.jax.org/app/browse/term/HP:0002894)
- pmid: PMID:25905195
- def: A type of neuroendocrine tumor that originates in the pancreas and secretes excessive amounts of vasoactive intestinal peptide (VIP).
- comment: Secretory diarrhea is the most characteristic symptom of a VIPoma. In severe cases, patients can produce up to 6-8L of watery stools per day.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9149
- synonyms: Vasoactive intestinal peptide-secreting tumor of the pancreas(NONE)
